signac_dashboard/views.py

```diff
@@ -65,7 +65,9 @@ def show_job(dashboard, jobid):
         return dashboard._render_job_view(default_view="grid")
 
 
-def get_file(dashboard, jobid, filename):
+def get_file(
+    dashboard, jobid, filename
+):  # not sure if *args, **kwargs needed in argument list
```

Comment:
The `_render_job_view` takes arguments `(self, *args, **kwargs)`. Not sure if `get_file` needs args and kwargs

Reply:
No. These functions serve different purposes.

`_render_job_view` is called by view functions like [`search`](https://github.com/glotzerlab/signac-dashboard/blob/c1af75af0fa0b27a1c55ffd924d3825d9dfb7649/signac_dashboard/views.py#L24) and [`jobs_list`](https://github.com/glotzerlab/signac-dashboard/blob/c1af75af0fa0b27a1c55ffd924d3825d9dfb7649/signac_dashboard/views.py#L44). It accepts `*args, **kwargs` because it [forwards those to the `render_template` function](https://github.com/glotzerlab/signac-dashboard/blob/c1af75af0fa0b27a1c55ffd924d3825d9dfb7649/signac_dashboard/dashboard.py#L385-L387). This was a compatibility layer once I realized that we needed to pass arguments like `default_view="list"` / `default_view="grid"` and it could be changed to `def _render_job_view(default_view)` now.

We don't need to accept arbitrary arguments here because `get_file` doesn't need to pass those onto anything else.